library/std/src/sys/windows/mod.rs

```diff
@@ -275,13 +275,13 @@ pub fn abort_internal() -> ! {
 cfg_if::cfg_if! {
     if #[cfg(target_vendor = "uwp")] {
         #[link(name = "ws2_32")]
-        // For BCryptGenRandom
-        #[link(name = "bcrypt")]
+        #[link(name = "bcrypt")] // For BCryptGenRandom
         extern "C" {}
     } else {
         #[link(name = "advapi32")]
```

Comment:
Now that we aren't using `RtlGenRandom`, we should add `// For OpenProcessToken` (as that's then only function left from `Advapi32.lib`).